### ISSUE TITLE
False positive in conditional binding cascade violation

### ISSUE THREAD

**Issue by ejensen** [realm/SwiftLint#642]
Conditional binding cascade violation warning is triggered when `let` and `var` is mixed. This is a false positive due to the fact that the type statements are required when the binding is heterogeneous.

ex:
``` swift
// triggers conditional_binding_cascade
guard let foo = someOptional(), var bar = someMutableOptional(foo) else {
  return
}
```

**Comment by norio-nomura** [comment]
Thanks for reporting! 🙏 
I pushed fix.

**Comment by norio-nomura** [comment]
For avoid filing duplicated issue, we should keep this issue opened until next release will be published.
